### ISSUE TITLE
Provide the option of raising a support ticket for Identify CLA Manager flow in case of no company admin

### ISSUE THREAD

**Issue by vinod-kadam** [communitybridge/easycla#2674]
Currently, when an organization has a company admin(s) then Contributor Console shows following screen for Identify CLA Manger flow:
![image.png](https://images.zenhubusercontent.com/5e9e9952b186e1325cf5608a/86b12090-fccd-4a0c-860f-833f01511f3e)
However, if the there is no company admin yet for an organization then following screens is shown:
![image.png](https://images.zenhubusercontent.com/5e9e9952b186e1325cf5608a/78b623ea-05b9-4102-885f-0775397f291a)

When there is no company admin then give the user to raise a support ticket.
Show following:

Please Click here to open a support ticket if you do not know who will be the CLA Manager.

"here" should be made the link: the link to use is https://jira.linuxfoundation.org/plugins/servlet/theme/portal/4/create/143.

**Comment by vinod-kadam** [comment]
This is deployed to DEV, Staging & PROD.

**Comment by vinod-kadam** [comment]
I checked it for an organization "Vinod IT Solutions" which doesn't have any company admin.
Following screen was shown which is correct.
 ![image.png](https://images.zenhubusercontent.com/5e9e9952b186e1325cf5608a/6e646903-4dab-4201-a022-4d724963fcf6)
On clicking "Click here", it opened the LFX EasyCLA JSD queue link in a new window which is correct,
![image.png](https://images.zenhubusercontent.com/5e9e9952b186e1325cf5608a/e1b2d1bb-8e9e-4707-861d-3c574d7666e5)

I checked it for an organization "Vinod Pharma" which has a company admin then it showed following screen which is correct.
![image.png](https://images.zenhubusercontent.com/5e9e9952b186e1325cf5608a/f35a7584-f155-4d06-b23c-2f918744fdc6)
